src/Identity/AwsCredentialIdentity.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Aws\Identity;
+
+/**
+ * Denotes the use of standard AWS credentials.
+ *
+ * @internal
+ */
+abstract class AwsCredentialIdentity implements IdentityInterface
+{
+    /**
+     * Returns a UNIX timestamp, if available, representing the expiration
```

Comment:
What does it do if one isn't available? Is this same conditional part of `BearerTokenIdentity` and not documented, or do those always have expirations?

The return type should also be noted.

Reply:
Updated. The annotations for both identity types should be identical